sw/cheri/link.ld

```diff
@@ -9,4 +9,43 @@ SECTIONS
 		*(.text .text.* .rodata .rodata.* .data.rel.ro);
 		. = ALIGN(4);
 	}
+
+    .data :
+    {
+        __DATA_BEGIN__ = .;
+        *(.data .data.*)
```

Comment:
Can you add a comment saying that the cheri compiler expects all the data sections together?